scripts/sv/fermiLite/scanBam.sh

```diff
@@ -36,5 +36,5 @@ KMER_KILL_LIST=$(echo "${REF_FASTA}" | sed 's/.fasta$/.kill.kmers/')
     --cluster "${CLUSTER_NAME}" \
     --num-executors 20 \
     --driver-memory 30G \
-    --executor-memory 30G \
+    --executor-memory 40G \
```

Comment:
Will change back. The reason for this change was to increase the memory so the 8-hanging jobs phenomenon could be temporarily solve by "brute force"